ports/gameinput/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "gameinput",
+  "version-date": "2024-04-29",
+  "description": "GameInput",
+  "homepage": "https://aka.ms/gameinput",
+  "license": null,
+  "supports": "windows & x64 & !uwp & !xbox"
```

```suggestion
  "version": "0.2303.22621.3038",
  "description": "GameInput",
  "homepage": "https://aka.ms/gameinput",
  "license": null,
  "supports": "windows & x64 & !uwp"
```

GameInput is available on PC and Xbox https://www.nuget.org/packages/Microsoft.GameInput

---

Technically the NuGet package only works on Windows. GameInput on Xbox is currently only available using the NDA Microsoft GDKX.

---

That said, I could make the `gameinput` port a kind of `'validation port' like 'ms-gdkx' for xbox.

---

I guess the other option is for *xbox*, I could find the Microsoft GDKX and copy `GameInput.h` and `GameInput.lib` if I can find it in the environment.